README.md

```diff
@@ -27,6 +21,18 @@ If you prefer to do it straight from CLI here is a one-liner for you:
 ```bash
 curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
 ```
+#### From source
+You can easily build OpenCompose tool from source. All you need is [Go installed](https://golang.org/dl/) on your system. Than simply run:
+```sh
+go get -u github.com/redhat-developer/opencompose
+```
+
+Although the simple `go get` installation works for just trying it out, if you want to submit bug reports you have to build it properly:
+```bash
+go get -u -d github.com/redhat-developer/opencompose
+make install -C $GOPATH/src/github.com/redhat-developer/opencompose/
+```
+This will make sure proper output of `opencompose version` and other features dependent on using linker flags.
```

Comment:
I wanted to avoid changing README if we decide to do any other advanced stuff in `make build` or `make install` but I'll rephrase it to be less advanced.